### PR TITLE
Cbrelease 4.8.15

### PR DESCRIPTION
**Pull request by NandiniAV**

### CODE REVIEW

File: project/ws/app/src/lib/routes/home/components/filter/filter.component.ts [hunk 1]
```diff
@@ -51,41 +52,46 @@ export class FilterComponent implements OnInit, AfterContentChecked {
   }
 
   ngOnInit() {
+    this.designationList = this.filterFacetsData.designation
```
Comment (vishnubansaltarento) [inline project/ws/app/src/lib/routes/home/components/filter/filter.component.ts]: Please do check if this.filterFacetsData exists then do assign

File: project/ws/app/src/lib/routes/home/components/filter/filter.component.ts [hunk 2]
```diff
@@ -51,41 +52,46 @@ export class FilterComponent implements OnInit, AfterContentChecked {
...
+        // if (data.from === 'content') {
+        //   this.getFilterEntity()
+        // this.getProviders()
+        // } else {
+        // if(this.usersSvc.trainingPlanAssigneeData &&
+        //   this.usersSvc.trainingPlanAssigneeData.category === 'Custom Users') {
+        //   this.getDesignation();
+        // }
+        if (this.designationList.length) {
+          //   this.getDesignation()
```
Comment (vishnubansaltarento) [inline project/ws/app/src/lib/routes/home/components/filter/filter.component.ts:73]: Do optional checks

File: project/ws/app/src/lib/routes/home/routes/users-view/users-view.component.ts [hunk 3]
```diff
@@ -217,6 +218,12 @@ export class UsersViewComponent implements OnInit, OnDestroy {
     reqBody = {
       request: {
         filters: filtreq,
+        facets: [
+          'profileDetails.professionalDetails.group',
```
Comment (vishnubansaltarento) [inline project/ws/app/src/lib/routes/home/routes/users-view/users-view.component.ts:222]: Check if profileDetails exists or not

File: project/ws/app/src/lib/routes/home/routes/users-view/users-view.component.ts [hunk 4]
```diff
@@ -228,6 +235,7 @@ export class UsersViewComponent implements OnInit, OnDestroy {
       this.activeUsersData = allusersData.content
       // this.activeUsersData = this.activeUsersData.filter((wf: any) => wf.profileDetails.profileStatus !== 'NOT-MY-USER')
       this.activeUsersDataCount = allusersData.count
+      this.filterFacets = allusersData.facets
```
Comment (vishnubansaltarento) [inline project/ws/app/src/lib/routes/home/routes/users-view/users-view.component.ts]: Same as above

Comment (NandiniAV) [reply project/ws/app/src/lib/routes/home/routes/users-view/users-view.component.ts]: done

File: project/ws/app/src/lib/routes/home/routes/users-view/users-view.component.ts (hunk 3 above)
Comment (NandiniAV) [reply project/ws/app/src/lib/routes/home/routes/users-view/users-view.component.ts:222]: this payload to be send can't add condition here

File: project/ws/app/src/lib/routes/home/components/filter/filter.component.ts (hunk 2 above)
Comment (NandiniAV) [reply project/ws/app/src/lib/routes/home/components/filter/filter.component.ts:73]: done

File: project/ws/app/src/lib/routes/home/components/filter/filter.component.ts (hunk 1 above)
Comment (NandiniAV) [reply project/ws/app/src/lib/routes/home/components/filter/filter.component.ts]: done

**Pull request merged**